include/cupla/cudaToCupla/runtime.hpp

```diff
@@ -88,4 +88,15 @@
 #define __fdividef(a,b) ((a)/(b))
 #define __expf(a) alpaka::math::exp(acc,a)
 #define __logf(a) alpaka::math::log(acc,a)
+#define __max(a,b) alpaka::math::max(acc,a,b)
+#endif
+
+#if !defined(ALPAKA_ACC_GPU_CUDA_ENABLED) && !defined(ALPAKA_ACC_GPU_HIP_ENABLED)
+#if defined(__GNUC__) || defined(__GNUG__)
+  #define __popc __builtin_popcount
```

Comment:
Maybe this is better as a function-style macro (taking an argument explicitly) for consistency with others?

Reply:
Yes we should use a function style macro!+1 for function style macro